channels/views/comments.py

```diff
@@ -161,6 +162,29 @@ def get_object(self):
         api = Api(user=self.request.user)
         return api.get_comment(self.kwargs['comment_id'])
 
+    def get(self, request, *args, **kwargs):
+        """GET a single comment and it's children"""
+        with translate_praw_exceptions():
+            comment = self.get_object()
+
+            # comment.refresh() raises if the comment
+            # isn't found
+            try:
+                comment.refresh()
+            except PRAWException:
```

Comment:
I didn't see a more particular one, unfortunately. I ran into the error when I was writing the '404' case test, and it was just throwing a `PRAWException`. The exception did have a more particular message attached to it (something like 'no data returned for comment' or something) but it seemed to be a general-case exception that was raised, instead of one particular to the 404 case